src/repror/cli/generate_html.py

```diff
@@ -44,19 +44,32 @@ def get_platform_fa(platform):
         return "fa-solid fa-question"  # Default emoji if platform is unknown
 
 
-def get_build_state_fa(build_state: BuildState):
-    if build_state == BuildState.SUCCESS:
-        return "fa-solid fa-check text-green-600"
+reproducible = "fa-solid fa-thumbs-up text-green-600"
+failure = "fa-solid fa-times text-red-600"
+non_reproducible = "fa-solid fa-thumbs-down text-red-300"
+
+
+def get_build_state_fa(
+    build_state: BuildState, rebuild_state: Optional[BuildState] = None
+):
+    if build_state == BuildState.SUCCESS and (
+        rebuild_state is None or rebuild_state == BuildState.SUCCESS
+    ):
+        return reproducible
```

Comment:
small qq:
if we dont have rebuild state, maybe we could have unknown state?
like the one at the end:
return "fa-solid fa-question"